docs/source/tutorial/03-write-your-first-query.mdx

```diff
@@ -2,51 +2,71 @@
 
-If you run this query by pressing the Play button in GraphQL Playground, the query returns results as a JSON object on the right-hand side of the page:
+<img src="images/explorer_launch_list_rename.png" class="screenshot" alt="Renaming the query"/>
 
-<img src="images/completed_basic_query.png" class="screenshot" alt="Query JSON in GraphiQL"/>
+Next, on the left hand side, you can select what fields you want back in the returned object. Start by clicking the button next to the `cursor` field. It will mark that field as selected, then insert it into your operations: 
 
-This query executes successfully, but it doesn't include any information about the `launches`! That's because we didn't include the necessary field in our query.
+<img src="images/explorer_check_cursor.png" class="screenshot" alt="After adding the cursor field."/>
 
-Update your query to fetch the `id` and `site` properties for each launch, like so:
+This is probably the easiest way to add fields to your object, since it knows how everything is spelled and what type everything is. 
 
-```graphql:title=(GraphiQL)
+However, you can also use auto-complete to help you with this. Add a newline below `cursor` in the Operations panel and start typing `ha`. An autocomplete box pops up and shows you options based on what's in the schema:
+
+<img src="images/explorer_autocomplete.png" class="screenshot" alt="Example of autocomplete"/>
+
+The Sandbox Explorer is a great tool for building and verifying queries so you don't have to repeatedly rebuild your project in Xcode to try out changes.
+
+As the schema indicates, the `launches` query returns a `LaunchConnection` object. This object includes a list of launches, along with fields related to pagination (`cursor` and `hasMore`). The query you've written so far indicates exactly which fields of this `LaunchConnection` object you want to be returned.
+
+Run this query by pressing the "Submit Operation" button, which should now have the name of your query, `LaunchList`: 
+
+<img src="images/explorer_submit_operation.png" class="screenshot" alt="Submit the operation"/>
+
+You'll quickly see the query returns results as a JSON object on the right-hand side of the page: 
+
+<img src="images/explorer_launch_list_initial_response.png" class="screenshot" alt="Query JSON in Sandbox Explorer"/>
+
+This query executes successfully, but it doesn't include any information about the `launches`! That's because we didn't include the necessary field in the query.
+
+Click the button next to the `launches` field at the bottom of the left column. It will add a set of braces for `launches` to the operations section, and then move the documentation to show information for the `Launch` type: 
```

Comment:
I know we don't really have a solution for that and that's completely out of scope for this PR but the `launches.launches` feels weird everytime. Could we rename the second `launches` to `nodes` maybe?

Reply:
In theory we could add an alias that renames it, otherwise that'd need to happen at the schema level. Definitely not worth fixing in this PR though.